### PR TITLE
Setup Fingerprinting and Metadata Tracking

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/13073/collect-metadata-on-mobile-sdk-v10-ios

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Networking/Models/v2/Metadata.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            return origin.rawValue
+        case .documentBackImageOrigin(let origin):
+            return origin.rawValue
+        case .cameraFacing(let facing):
+            return facing.rawValue
+        }
+    }
+    
+    private enum CodingKeys: String, CodingKey {
+        case type, value
```
Comment (vanshg) [inline Sources/SmileID/Classes/Networking/Models/v2/Metadata.swift:38]: It should be `name` and value

**Comment by vanshg** [comment]
Can you point this to a new base branch? We don't have the backend set up to accept these values yet

File: Sources/SmileID/Classes/SmileID.swift [hunk 2]
```diff
@@ -57,6 +59,12 @@ public class SmileID {
         self.useSandbox = useSandbox
         self.apiKey = apiKey
         SmileIDResourcesHelper.registerFonts()
+        Task {
+            // we use the deviceId and not fingerprint as this is unique and does not change
```
Comment (vanshg) [inline Sources/SmileID/Classes/SmileID.swift:63]: Will device ID return the same value in different apps? This is not the case on Android so I decided to use fingerprint there

**Review by vanshg — APPROVED** [review]
LGTM pending 1 question

File: Sources/SmileID/Classes/SmileID.swift (hunk 2 above)
Comment (jumaallan) [reply Sources/SmileID/Classes/SmileID.swift:63]: Yes it does. The iOS docs say deviceId is unique but fingerprint changes even with os updates
